gc/base/SweepPoolManagerAddressOrderedListBase.cpp

```diff
@@ -236,10 +245,18 @@ MM_SweepPoolManagerAddressOrderedListBase::connectChunk(MM_EnvironmentBase *env,
 						previousConnectChunk->trailingFreeCandidate);
 
 				connectChunkPostProcess(chunk, sweepState, (MM_HeapLinkedFreeHeader *)previousConnectChunk->trailingFreeCandidate, previousFreeEntry);
+
+				if (memoryPool->doesNeedClearCardForConnectFreeEntry()) {
+					memoryPool->cleanCardsForFreeEntry(env, (void*)previousFreeEntry, ((uint8_t*)previousFreeEntry + previousFreeEntrySize), false);
```

Comment:
I already commented in MPAOL class, but all cleanCardsForFreeEntry (and now doesNeedClearCardForConnectFreeEntry, setNeedClearCardForConnectFreeEntry, and possibly some others) do not belong to MPAOL, but to SweepPoolManagerVLHGC. If they are called only from SweepPoolManagerVLHGC variant of connectChunkPostProcess, we don't need them even in MM_SweepPoolManagerAddressOrderedListBase.